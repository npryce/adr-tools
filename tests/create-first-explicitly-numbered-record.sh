adr new -n 52 The First Decision
cat doc/adr/0052-the-first-decision.md
