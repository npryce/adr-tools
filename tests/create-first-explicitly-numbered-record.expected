adr new -n 52 The First Decision
doc/adr/0052-the-first-decision.md
cat doc/adr/0052-the-first-decision.md
# 52. The First Decision

Date: 1992-01-12

## Status

Accepted

## Context

The issue motivating this decision, and any context that influences or constrains the decision.

## Decision

The change that we're proposing or have agreed to implement.

## Consequences

What becomes easier or more difficult to do and any risks introduced by the change that will need to be mitigated.
